pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/entity/display/transform/TransformBuilder.kt

```diff
@@ -118,6 +118,6 @@ class TransformBuilder(val components: ArrayDeque<TransformComponent>) {
     }
 
     companion object {
-        private val BLOCK_DISPLAY_ADJUSTMENT: Vector3f = Vector3f(-0.5F)
+        private val BLOCK_DISPLAY_ADJUSTMENT: Vector3f = Vector3f(-0.5F, -0.5F, -0.5F)
```

Comment:
same comment is on entities PR, I think plugins would find having this value useful, so
```suggestion
        @JvmField
        val BLOCK_DISPLAY_ADJUSTMENT: Vector3f = Vector3f(-0.5F, -0.5F, -0.5F)
```